app/konnect/api-products/index.md

```diff
@@ -28,8 +26,10 @@ Here are some of the things you can do from the API Products Dashboard:
 -------|------|------------
 **Overview** | Analytics data for the API product. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/).
 **Product Versions** | This section displays the status of an API product version. From the context menu you can **Delete** an API product version, or use the **View Details** button to navigate to that version's dashboard. 
-**Documentation** | You can add markdown documentation for your API product, as well as an API specification for each version of the API product. You can control the individual publishing status of each document you upload to a API product.
+**Documentation** | You can add and edit markdown documentation for your API product, upload API specs and manage the publishing status for each document. 
 
+{:.note}
+> **Note**: API products can only be published to the Dev Portal within your [geographic region](/konnect/geo). To publish to a Dev Portal in a different region, switch geos using the control in the bottom-left corner of {{site.konnect_product_name}}.
```

Comment:
```suggestion
> **Note**: API products can only be published to the Dev Portal in the [geographic region](/konnect/geo) you currently have selected. To publish to a Dev Portal in a different region, switch geos using the control in the bottom-left corner of {{site.konnect_product_name}}.
```
Maybe? I just wanted to make it clear it's the one you've selected, not where the admin is physically located.